src/main/pages/che-7/installation-guide/proc_configuring-namespace-strategies.adoc

```diff
@@ -1,27 +1,21 @@
 [id="configuring-namespace-strategies_{context}"]
 = Configuring namespace strategies
 
-This paragraph is the procedure module introduction: a short description of the procedure.
+Word 'namespace' is used interchangeably with OpenShift's 'project'.
 
-[discrete]
-== Prerequisites
+=== Namespace per Workspace strategy
+TODO
```

Comment:
Let's not put empty sections with 'TODO' into published docs. Instead, I suggest leaving out the section and filing an issue for adding it later.